bitnami/redmine/Chart.yaml

```diff
@@ -1,6 +1,6 @@
-apiVersion: v1
```

Comment:
You need to follow this process for the standardization:
```console
$ cat requirements.yaml >> Chart.yaml
$ rm -rf requirements.*
$ sed -i "s@apiVersion: v1@apiVersion: v2@g" Chart.yaml
$ yq r --tojson Chart.yaml | jq . | yq r --prettyPrint - > Chart.yaml.new
$ mv Chart.yaml.new Chart.yaml
```

I think some of the commands are already done, but the latest two commands were not executed
```console
$ yq r --tojson Chart.yaml | jq . | yq r --prettyPrint - > Chart.yaml.new
$ mv Chart.yaml.new Chart.yaml
```

In the end, we need to see the fields in this file alphabetically ordered

Reply:
I think the command is missing the `-S` flag in order to sort fields then 🙂:

```bash
$ yq r --tojson Chart.yaml | jq -S . | yq r --prettyPrint - > Chart.yaml.new
```

Reply:
The sort must be done as part of one of the other commands since I used the above command (without `-S`) to generate the _Chart.yaml_ in the different PRs and the fields were sorted. Maybe it depends on the `yq`/`jq` version or something